pkgs/development/libraries/igraph/default.nix

```diff
@@ -29,7 +53,7 @@ stdenv.mkDerivation rec {
   meta = {
     description = "The network analysis package";
     homepage = "https://igraph.org/";
-    license = lib.licenses.gpl2;
+    license = lib.licenses.gpl2Plus;
     # NB: Known to fail tests on aarch64.
     platforms = [ "x86_64-linux" ] ++ lib.platforms.darwin;
```

Comment:
Can you mark it broken for aarch and set the arch to either x86_64 only or all?

Reply:
I actually want to make it run on aarch64. There's an upstream issue: https://github.com/igraph/igraph/issues/1694